src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,33 +1,63 @@
 package edu.kis.vh.nursery;
 
+/**
+ * @author Jan
+ * 
+ */
 public class DefaultCountingOutRhymer {
 
-	private int[] numbers = new int[12];
+	private static final int STACK_SIZE = 12;
+	private static final int FULL_COUNT = 11;

```

Comment:
This constant is related to STACK_SIZE . Changing only one leads to an error (inconsistency).